### ISSUE TITLE
BY-51  As a developer, I want to research how to have recent searches of places to appear when a user initiates a search for a new location.

### ISSUE THREAD

**Issue by JenYn** [jaychenblue/scout-concordia#73]
List of recently searched places appears when looking up locations in the search bar.

This feature was requested by the product owner, more research needs to be done before making a decision regarding whether or not we will implement it.

**Comment by JenYn** [comment]
This is a continuation of BY-58 [#81 ].

Users should be able to see their recently searched locations when they tap on the search bar

Checklist of features that should be included for this issues to be considered completed:

[] When the user taps on the search bar, a list of recently searched locations is displayed
[] If one of the displayed items is selected, the map will zoom and center in on the location when the user hits enter
[] if the user clicks on the 'x', the location is erased. 

**Once these features have been completed, this issue can be moved to the "In review" Pipeline.

**Comment by JenYn** [comment]
The final result should resemble this mockup:

![Mockups Final Final-05 (1)](https://user-images.githubusercontent.com/26283932/75593699-2b64cb00-5a54-11ea-9bae-e54bc8af6c8c.png)

**Comment by JenYn** [comment]
**Implementation strategy:**

The following resources were found, which will help guide us with the implementation:

- https://stackoverflow.com/questions/33528387/android-get-recent-searches-from-places-api

- https://proxus.dk/Blog/Post/2014/12/19/android-autocomplete-edittext-with-history

**Comment by JenYn** [comment]
This feature was requested by the product owner as a low priority. After researching how to implement it, we realized that we need to first understand and set up local storage, thus we will discuss the importance of this feature with the product owner in order to decide whether we will implement it during print #4 (since it is low priority), or drop it entirely due to it not being a high priority.
